i18n/locales/en/common.json

```diff
@@ -146,6 +148,8 @@
   "Edit bookmark": "Edit bookmark",
   "Edit transaction": "Edit transaction",
   "Edit voting": "Edit voting",
+  "Eg. 1,5": "Eg. 1,5",
+  "Eg. 10180477": "Eg. 10180477",
```

Comment:
https://github.com/LiskHQ/lisk-hub/pull/2526#discussion_r331991847 and https://github.com/LiskHQ/lisk-hub/pull/2526/commits/89538d4fca12bf50971850f00f7f03f5648ceb27 could be applied here, too.

Reply:
But doesn't it make sense to translate these placeholders too? The way to express things like "e.g." and "i.e." are different in other languages, including German @yasharAyari

Reply:
@colado I think you misunderstood what should be done, please check the links above again.

Reply:
Got it! :) Changed now.